README.adoc

```diff
@@ -252,9 +231,11 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see if the tests detect a failure, comment out the `changeSystemProperty()` methods
-in the `FaultToleranceTest.java` file. Rerun the Maven build.
+To see if the tests detect a failure, comment out the `@Fallback` annotation
+in the `src/main/java/io/openliberty/guides/inventory/InventoryManager.java` file. Rerun the Maven build.
 You see a test failure occur for the `testFallbackForGet()` test case.
+The test failure indicates a 500 HTTP response code because the fallback mechanism was not in place to action
+when the IOException was thrown.
```

Comment:
We have discussed that we need more to address #44.  Just noting it.

Reply:
Ok.